src/org/zalando/stups/kio/api.clj

```diff
@@ -75,7 +84,9 @@
         is-human? (= "/employees" realm)
         has-scope? (set (from-token request "scope"))]
     (if is-human?
-      (when-not has-auth?
+      (when-not (or
+                  has-auth?
+                  (admin-uid? request))
```

Comment:
I would suggest to check the admin role even earlier. `(auth/get-auth request team)` performs an http request to the magnificent api. This can be avoided in case of admins